internal/commands/extension_inspect.go

```diff
@@ -24,18 +28,39 @@ func ExtensionInspect(logger logging.Logger, cfg config.Config, client PackClien
 			extensionName := args[0]
 			registry := flags.Registry
 			if registry == "" {
-				// fix registry for extension
+				registry = cfg.DefaultRegistryName
 			}
 
 			return extensionInspect(logger, extensionName, registry, flags, cfg, client)
 		}),
 	}
-	// flags will be added here
+	cmd.Flags().IntVarP(&flags.Depth, "depth", "d", -1, "Max depth to display for Detection Order.\nOmission of this flag or values < 0 will display the entire tree.")
+	cmd.Flags().StringVarP(&flags.Registry, "registry", "r", "", "buildpack registry that may be searched")
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&flags.Registry, "registry", "r", "", "extension registry that may be searched")
```
(or maybe just omit it for now, since there is no extension registry :))

Reply:
Yes I've made the changes locally, didn't push yet as wrapping up all the work at once, previous pushes was just to clean up the git history due to merges